### DISCUSSION TITLE
post_233662

### DISCUSSION THREAD

**Discussion by cincuranet** [cincuranet/blog-comments#1276] [Comments]
imported 

https://www.tabsoverspaces.com/id/233662

**Comment by cincuranet** [comment]
Imported comment written by _Mateusz_ on _2020-04-13T19:50:03_:

<p>Old but still valuable. Thanks!</p>

**Comment by ghost** [comment]
Proved useful in the context of reproducible build and CI/CD.

In that context, I needed the SHA256 of the output binaries to remain the same across re-builds.

Even though I used the reproducible switches for MSbuild and I embedded the pdb file in the output binary (dll), it still contained a reference to the pdb file with its path, which contained the random build root folder, throwing off the sha256.
With folder mapping, the path is fixed, which solves the issue.

**Comment by FaltoGH** [comment]
Thanks a lot!
